cypress/fixtures/adminlogin.js

```diff
@@ -0,0 +1,21 @@
+[
+    {
+      action: "type",
```

Comment:
Can you name your files in a consistent case. this file is in just lower case even tho the others in the same folder are in camelCase